src/utils/api-requests.test.ts

```diff
@@ -6,7 +6,18 @@ jest.mock('axios');
 const url = 'https://example.com';
 const method = 'get';
 
-describe('apiRequest', () => {
+describe('utils/api-requests.ts', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('handles null parameters', async () => {
```

Comment:
When would this happen? Is it a valid case, or should it be resolved at a higher level?

Reply:
We do handle in most cases checking that the url is not null higher up in the calling business logic.

Perhaps this is overkill.